data/core/init.lua

```diff
@@ -535,16 +535,18 @@ end
 
 function core.load_plugins()
   local no_errors = true
-  for _, root_dir in ipairs {DATADIR, USERDIR} do
+  local loaded = {}
```

Comment:
I do not understand why the "loaded" check here is needed since we load the plugin with "require" and it does already keep trace of what is already loaded. I am missing something ?

Reply:
Yeah, removing it. Not sure what I was thinking when I added it.